examples/annotation_import/image.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Something is wrong with these links they aren't rendering properly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The image renders properly in the colab notebook, is it possible this is an issue with app.reviewnb? </p>

Reply:
yeah I think so, I realized this in a different review.